tests/llmobs/conftest.py

```diff
@@ -97,6 +97,7 @@ def mock_writer_logs():
 def mock_evaluator_logs():
     with mock.patch("ddtrace.llmobs._evaluators.runner.logger") as m:
         yield m
+        m.reset_mock()
```

Comment:
this shouldn't be necessary, since the fixture is being reset after yielding anyway